content/guides/nextjs/github-open-authoring/hosting-vercel.md

```diff
@@ -1,46 +1,42 @@
 ---
 title: Hosting with Vercel
 ---
-
 Finally, we're ready to get this app in production. Make sure all the latest changes are pushed up to your repository. These are the steps to getting the `create-next-app` up and running on Vercel.
 
-1. **Go to https://vercel.com/ and signup/login.**
-
-2. **On your dashboard click "Import Project".**
-
+1. **Go to** [**https://vercel.com**](https://vercel.com)**/ and** [**signup**](https://vercel.com/signup) **or** [**login**](https://vercel.com/login?next=%2Fdashboard)**.**
+2. **On your dashboard click** [**Import Project**](https://vercel.com/import)**.**
    1. Under "From Git Repository" click Continue.
-   2. Connect to GitHub and click "Import Project from Github".
-   3. Select your repository.
-   4. Leave the root directory empty.
-   5. It will auto-detect that you're using Next.js. The default build, output, and development commands don't need to be changed.
-   6. Click "Deploy".
+   2. Enter the URL of your repository on GitHub 
+   3. Leave the root directory empty.
+   4. It will auto-detect that you're using Next.js. The default build, output, and development commands don't need to be changed.
+   5. Click "Deploy".
 
 > Note this **deploy will fail** since environment variables aren't set for the GitHub Client yet.
 
-3. **Create a new OAuth App** pointing to your Vercel deployment instead of localhost. Go back to the [Github OAuth page](https://github.com/settings/developers) in your Developer Settings. Your site will be visible from a generated domain — typically _your-site-name.now.sh_ — use this as the base URL. This will be the _production app_, we can add the client id and client secret values to our environment variables in Vercel.
+1. [**Create a new GitHub OAuth App**]() pointing to your Vercel deployment instead of localhost. Go back to the [Github OAuth page](https://github.com/settings/developers) in your Developer Settings. Your site will be visible from a generated domain — typically _your-site-name.vercel.app_ — use this as the base URL. This will be the _production app_, we can add the client id, client secret, and signing key values to our environment variables in Vercel.
```

Comment:
These list item numbers got twisted up it seems. And an empty link was added here.

Reply:
I've tried to fix those, forcing myself to only use Tina here. 😅